cuberite/snapcraft.yaml

```diff
@@ -28,6 +28,9 @@ parts:
         build-packages:
             - gcc
             - g++
+        snap:
+            - -include
```

Comment:
These are the libjsoncpp headers, which are the only thing that gets installed through `make install`. So they shouldn't be shipped as part of the package.